tests/scripts/anchor-mardown-headings.test.js

```diff
@@ -5,6 +5,8 @@ const test = require('tape')
 test('anchorMarkdownHeadings', (t) => {
   const anchorMarkdownHeadings = require('../../scripts/plugins/anchor-markdown-headings')
 
+  t.timeoutAfter(500)
```

Comment:
Will this have any effect as 500ms is the default already? https://github.com/substack/tape#testname-opts-cb